recorder.go

```diff
@@ -37,10 +38,16 @@ const (
 	// between child and parent spans.
 	ParentSpanGUIDKey = "parent_span_guid"
 
-	ComponentNameKey = "component_name"
-	GUIDKey          = "guid" // <- runtime guid, not span guid
-	HostnameKey      = "hostname"
-	CommandLineKey   = "command_line"
+	TracerPlatformValue = "go"
+	TracerVersionValue  = "0.9.1"
```

Comment:
"would be nice" to have an update mechanism for this, but whatever (perhaps a TravisCI/CircleCI thing that bumps the patch version with every merge to master). totally understand if that's not above the bar for things that actually get done :)